x/params/go.mod

```diff
@@ -0,0 +1,136 @@
+module cosmossdk.io/x/params
+
+go 1.21.0
+
+require (
+	cosmossdk.io/api v0.7.1
+	cosmossdk.io/core v0.12.0
+	cosmossdk.io/depinject v1.0.0-alpha.4
+	cosmossdk.io/errors v1.0.0
+	cosmossdk.io/log v1.2.1
+	cosmossdk.io/math v1.1.2
+	cosmossdk.io/store v1.0.0-rc.0
+	github.com/cometbft/cometbft v0.38.0
+	github.com/cosmos/cosmos-db v1.0.0
+	github.com/cosmos/cosmos-proto v1.0.0-beta.3
+	github.com/cosmos/cosmos-sdk v0.47.5
```

Comment:
```suggestion
	github.com/cosmos/cosmos-sdk v0.51.0
```

Reply:
Unknown revision apparently, neither has v0.50.0 been cut yet, just the betas and release candidates per https://github.com/cosmos/cosmos-sdk/releases

Reply:
Yes but modules on main have replaces of the SDK, so we should still add that.